site/pages/institution-en.hbs

```diff
@@ -8,21 +8,19 @@
 		{ "title": "Home", "link": "https://www.canada.ca/en.html" },
 		{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
 	],
-	"dateModified": "2014-10-24",
+	"dateModified": "2017-09-29",
 	"share": "true"
 }
 ---
 <div class="row">
-	<div class="col-sm-7 pull-left">
-		<p>We are a federal institution that is part of the <a href="#">[portfolio name] portfolio</a>.</p>
-		<p>[Institution name] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor.</p>
-	</div>
+<div class="col-sm-7 pull-left">
+<p>[Institution name] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor.</p>
+</div>
 	<div class="clr-rght-sm clr-rght-md clr-rght-lg"></div>
 	<div class="col-sm-5 pull-right">
```

Comment:
Can you do that block without the "pull-". Try to use instead "col-sm-pull-" and "col-sm-push-"